chunk/download.go

```diff
@@ -67,6 +69,9 @@ func (d *Downloader) thread() {
 
 func (d *Downloader) download(client *http.Client, req *Request, buffer []byte) {
 	Log.Debugf("Starting download %v (preload: %v)", req.id, req.preload)
+	if d.acknowledgeAbuse {
+		req.acknowledgeAbuse = true
```

Comment:
You shouldn't modify the Request struct passed as input here, instead set `acknowledgeAbuse` when the struct ist built in `requestChunk()` of the manager.

Reply:
resolved via 441ad17d0defde090917b837265f6ce06aea80e9